utils/testmap.py

```
@@ -0,0 +1,142 @@
+
+# 2. Interpolation and mapping
+def create_interpolated_overlay(data, resolution=1000):
+    lats = np.array([d[0] for d in data])
+    lons = np.array([d[1] for d in data])
+    pms = np.array([d[2] for d in data])
+    aqis = np.array([pm25_to_aqi(pm) for pm in pms])
+
+    lat_min, lat_max = lats.min(), lats.max()
+    lon_min, lon_max = lons.min(), lons.max()
+
+    grid_lat, grid_lon = np.mgrid[lat_min:lat_max:complex(resolution), lon_min:lon_max:complex(resolution)]
```

Pylint (3.3.7)

[convention] Line too long (107/100)

(C0301)